### PR TITLE
No More Carts!

### PR DESCRIPTION
**Pull request by dsmith328**
## About The Pull Request
Stops Abnos from being put into, or moved onto, anything that's a subtype of a "closet"

## Why It's Good For The Game
While genuinely funny as fuck, it can lead to some... issues. See: Someone moving Scorched Girl into EO's office so it's permanently there until the next breach, where it'll inevitably kill said EO.

## Changelog
:cl:
remove: Removed Super Abnormality Cart 3
/:cl:

### CODE REVIEW

**Comment by EgorDinamit** [comment]
This is already impossible due to abnormalities having huge mob_size, they won't fit in the closets, unless someone changed their size.

**Comment by dsmith328** [comment]
> This is already impossible due to abnormalities having huge mob_size, they won't fit in the closets, unless someone changed their size.

https://discord.com/channels/982658747304378520/982658750747926616/1017623980439519332

**Comment by EgorDinamit** [comment]
From what I gathered, the issue is drag&drop. Simply fix it.

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 1]
```diff
@@ -341,11 +341,13 @@
 		"<span class='warning'>You [actuallyismob ? "try to ":""]stuff [O] into [src].</span>", \
 		"<span class='hear'>You hear clanging.</span>")
 	if(actuallyismob)
+		var/mob/living/L = O
+		if(L.mob_size > max_mob_size)
+			return
```
Comment (EgorDinamit) [inline code/game/objects/structures/crates_lockers/closets.dm:346]: Provide a message that says why it didn't work.

Comment (dsmith328) [reply code/game/objects/structures/crates_lockers/closets.dm:346]: image

**Comment by dsmith328** [comment]
image
Added message

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 2]
```diff
@@ -341,11 +341,16 @@
 		"<span class='warning'>You [actuallyismob ? "try to ":""]stuff [O] into [src].</span>", \
 		"<span class='hear'>You hear clanging.</span>")
 	if(actuallyismob)
+		var/mob/living/L = O
+		if(L.mob_size > max_mob_size)
+			user.visible_message("<span class='notice'>But [L] is too big to fit into [src]!</span>", \
+			"<span class='notcing'>[L] is too big for you to fit into [src]!</span>", \
```
Comment (EgorDinamit) [inline code/game/objects/structures/crates_lockers/closets.dm:347]:
```suggestion
			"<span class='notice'>[L] is too big for you to fit into [src]!</span>", \
```

**Pull request merged**